maintainers.rst

```diff
@@ -30,6 +30,11 @@ Armv7-A architecture port
 :M: Etienne Carriere <etienne.carriere@linaro.org>
 :G: `etienne-lms`_
 
+Arm System Guidance for Infrastructure / Mobile FVP platforms
+-------------------------------------------------------------
+:M: Nariman Poushin <nariman.poushin@linaro.org>
+:G: `nariman`_
```

Comment:
Can you add the paths to the folders that you are going to be the maintainer of?

Reply:
Done

Reply:
What about ``plat/arm/board/{sgi575,sgm775}``?

Reply:
Of course, apologies for omitting. Fixed.